bitmark-references/bitmark-node-software/bitmark-blockain.md

```diff
@@ -0,0 +1,125 @@
+# Blockchain
+
+## Block
+
+Block is a bunch of transactions that have been added to blockchain. A bitmark block contains data as follows:
+
+| size (bytes) | data                                                           |
+|--------------|----------------------------------------------------------------|
+|            2 | block version number                                           |
+|            2 | transaction count inside the block                             |
+|            8 | block number (block height)                                    |
+|           32 | argon2 hash of previous block header                           |
+|           32 | sha3 hash of all transactions in the block (merkle root)       |
+|            8 | timestamp of block generation time, since 1970-01-01T00:00 UTC |
+|            8 | current difficulty in compact mode                             |
+|            8 | number to make hash meets difficulty                           |
+|   ... to end | transactions                                                   |
+
+Different types of transaction includes different lenegh and content inside.
+
+### Block Verification
+
+When a block is received, following checks will be made:
```

Comment:
I think should be `the following checks`.

Reply:
changed